src/debugger/sourceMap.ts

```diff
@@ -94,8 +100,8 @@ export class SourceMapUtil {
      * 1. It changes the path from absolute to be relative to the sourcesRootPath parameter.
```

Comment:
This JSDoc doesn't reflect method's functionality anymore. Could you please update